smozROM.py

```diff
@@ -38,37 +38,7 @@ def debug(str):
     if DEBUG: print(str)
 
 def printHeader():
-    print("\n  SmozROM Hack Applier v1.0.1 - smozoma 2018-10\n")
-
-def clearScreen(): 
-    if os.name == 'nt':
-        os.system('CLS')
-    else:
-        os.system('clear')
-
-def wait_key():
```

Comment:
These functions need to be restored or it won't work (clearScreen and wait_key)

Reply:
addressed in new pull request.